manifests/n/NicoJeskeIT/MasterVolumeSync/1.0.7/NicoJeskeIT.MasterVolumeSync.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NicoJeskeIT.MasterVolumeSync
+PackageVersion: 1.0.7
+PackageLocale: en-US
+Publisher: Nico Jeske IT
+PackageName: Master Volume Sync

```

```suggestion
PackageName: MasterVolumeSync
PackageUrl: https://github.com/nicojeske/MasterVolumeSync
```
Should match Control Panel / Apps&Features / Registry's `DisplayName` key as WinGet relies on the metadata from registry for matching upgrades.

Also suggested additional metadata.